### PR TITLE
Settings UI: remove disconnect link in footer. Add it in Debugger page.

### LINKED ISSUE #6219: Remove footer disconnect link
To keep flows consistent:

- [x] Remove the "Disconnect Jetpack" link from the footer of the UI
- [x] On the "Debug" page, add a "Disconnect from WordPress.com" link

The preferred way of disconnecting will be through the main UI (General tab within settings).

screen shot 2017-01-27 at 3 54 17 pm
screen shot 2017-01-27 at 3 55 41 pm

### PR DESCRIPTION
**Pull request by eliorivero**
Fixes #6219 

#### Changes proposed in this Pull Request:
- Settings: remove disconnect link in footer.
- Debug: add text about connection and link to disconnect Jetpack.

After the disconnection a notice shows up stating that the site was successfully disconnected and prompting user to go to connection screen.

#### Testing instructions:
* verify that there is no link to disconnect Jetpack in the footer of the settings page
* verify that there's a link in the `admin.php?page=jetpack-debugger` page and that it disconnects Jetpack

### CODE REVIEW

File: class.jetpack-debugger.php [hunk 1]
```diff
@@ -243,6 +255,24 @@ public static function jetpack_debug_display_handler() {
 							<li>- <?php esc_html_e( "If you get a 404 message, contact your web host. Their security may block XMLRPC.", 'jetpack' ); ?></li>
 						</ul>
 					</li>
+					<?php if ( current_user_can( 'jetpack_disconnect' ) && Jetpack::is_active() ) : ?>
+						<li>
+							<strong><em><?php esc_html_e( 'A connection problem with WordPress.com.', 'jetpack' ); ?></em></strong>
+							<?php
+							echo wp_kses(
+								sprintf(
+									__( 'Jetpack works by connecting to WordPress.com for a lot of features. Sometimes that connection gets messed up, and you need to disconnect and reconnect to get things working again. <a href="%s" class="jetpack-disconnect-site">Disconnect from WordPress.com</a>, then reconnect and see if that helps.', 'jetpack' ),
```
Comment (MichaelArestad) [inline class.jetpack-debugger.php]: I would change it to say:

```
Jetpack works by connecting to WordPress.com for a lot of features. Sometimes, when the connection gets messed up, you need to disconnect and reconnect to get things working properly. <a href="%s" class="jetpack-disconnect-site">Disconnect from WordPress.com</a>
```

**Comment by MichaelArestad** [comment]
Tested the disconnection out. Works like a charm. Is there any way we can reconnect without sending the user to an additional page?

Options:
* send the user straight to the connection page after clicking disconnect
* reconnect link right from the banner

**Comment by eliorivero** [comment]
Works now by redirecting user to Jetpack admin after disconnection so user can disconnect.

**Comment by MichaelArestad** [comment]
:shipit:

**Pull request merged**